### PR TITLE
Support multiple themes in the settings

### PR DESCRIPTION
**Pull request by Amalkh5**
Support multiple themes in the settings

Screenshots for  some of the change:

unnamed

unnamed 1

unnamed

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @Amalkh5 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/Amalkh5/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Phimp.me is a Photo App for Android that aims to replace proprietary photo applications. It offers features such as taking photos, adding filters, editing images and uploading them to social networks.
 
+Phimp.me is a Photo App for Android that aims to replace proprietary photo applications. It offers features such as taking photos, adding filters, editing images and uploading them to social networks.
```
Comment (Abhi2424shek) [inline README.md]: Duplicate text please see the above paragraph or point. Is'nt this is an unwanted addition?

File: README.md [hunk 2]
```diff
@@ -75,6 +77,62 @@ Settings|Add Accounts|Not working
  |Local photos : choose folder|Not working
 ## Development Setup
 
+	 It is advisable to clone only the development branch using following command:
```
Comment (Abhi2424shek) [inline README.md]: Please don't do unnecessary things ;)

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 3]
```diff
@@ -186,6 +190,10 @@
     //private GestureDetector gestureScanner;
     //View.OnTouchListener gestureListener;
     public static int width, height;
+    public final static int THEME_dark = 2;
+
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Blank lines.

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 4]
```diff
@@ -1007,4 +1028,4 @@ public int getCount() {
             return 5;
         }
     }
-}
+}
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Please don't remove the newline at EOF.

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 5]
```diff
@@ -195,6 +203,19 @@ public void onCreate(Bundle savedInstanceState) {
...
         ctx = this;
+
+        //set dark theme
+
+        if(Utility.getTheme(getApplicationContext()) == THEME_dark ) {
+        setTheme(R.style.AppTheme_Dark);}
+
+
+
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Blank lines.

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 6]
```diff
@@ -195,6 +203,19 @@ public void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
         ctx = this;
+
+        //set dark theme
+
+        if(Utility.getTheme(getApplicationContext()) == THEME_dark ) {
+        setTheme(R.style.AppTheme_Dark);}
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Please properly intend the code with 4 spaces.

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 7]
```diff
@@ -186,6 +190,10 @@
     //private GestureDetector gestureScanner;
     //View.OnTouchListener gestureListener;
     public static int width, height;
+    public final static int THEME_dark = 2;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Use camelCase for variable names.

File: app/src/main/java/vn/mbm/phimp/me/Settings.java [hunk 8]
```diff
@@ -80,11 +82,14 @@
 import vn.mbm.phimp.me.utils.RSSUtil;
 
 import static android.os.Environment.getExternalStorageDirectory;
+import static com.facebook.FacebookSdk.getApplicationContext;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Why was this done? Curious

File: app/src/main/java/vn/mbm/phimp/me/Settings.java [hunk 9]
```diff
@@ -434,8 +443,40 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked)
...
+    private void changeTheme() {
+
+        radiotDarkBtn = (RadioButton) getView().findViewById(R.id.radiotDarkBtn);
+        radiotLightBtn = (RadioButton)getView().findViewById(R.id.radiotLightBtn);
+
+        if(Utility.getTheme(getApplicationContext()) == PhimpMe.THEME_dark) {
+            radiotLightBtn.setChecked(false);
+            radiotDarkBtn.setChecked(true);
+
+        }else  {
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Add only one space after else

File: app/src/main/java/vn/mbm/phimp/me/Settings.java [hunk 10]
```diff
@@ -825,4 +866,11 @@ public void onClick(DialogInterface dialog, int which)
...
 
-}
+
+	private void recreateActivity() {
+		Intent i = getApplicationContext().getPackageManager()
+				.getLaunchIntentForPackage(getApplicationContext().getPackageName() );
+		i.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK );
+		startActivity(i);
+	}
+}
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Add a newline at eof

**Comment by Amalkh5** [comment]
@Abhi2424shek please review. Made the necessary changes.

File: app/src/main/java/vn/mbm/phimp/me/Settings.java (hunk 8 above)
Comment (Amalkh5) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: is there any problem or any thing that i need to change ?

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Do we really need to use this link?

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: * lib not link

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: @Amalkh5 ^

Comment (Amalkh5) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: @Abhi2424shek yes for this line getApplicationContext()

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: Ok

File: app/src/main/java/vn/mbm/phimp/me/PhimpMe.java [hunk 11]
```diff
@@ -65,7 +65,7 @@
 @SuppressWarnings("deprecation")
 public class PhimpMe extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener //, android.view.GestureDetector.OnGestureListener
 {
-    public static Context ctx;
+    public static Context ctx ;
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/PhimpMe.java]: Remove the trailing white space

**Comment by Amalkh5** [comment]
@Abhi2424shek I want to ask about the conflicts files , what the problem with them ? i just add new color for the dark theme

**Comment by Abhi2424shek** [comment]
I don't know they seems to be complex to view in web so I am sorry (I am on mobile now). You can resolve the conflicts by pulling from the upstream and making a commit after the changes have been made.

**Comment by Amalkh5** [comment]
@Abhi2424shek  every thing is good know , no conflicts

**Comment by Amalkh5** [comment]
@Abhi2424shek I did what you said, and now everything is good

**Comment by mariobehling** [comment]
@Amalkh5 Thank you. Please help to resolve the merge conflicts that appeared now. Thank you.

**Comment by Abhi2424shek** [comment]
Next time please do open an issue for it! Merging this one.

**Pull request merged**